packages/lesswrong/server/repos/PostsRepo.ts

```diff
@@ -42,6 +43,22 @@ export type PostAndCommentsResultRow = {
   last_commented: Date|null
 };
 
+const constructFilters = ({ startDate, endDate, minKarma, showEvents }: FilterPostsForReview): [string, Record<string, any>] => {
+  const params = {
+    ...(startDate && { startDate: startDate.toISOString() }),
+    ...(endDate && { endDate: endDate.toISOString() }),
+    ...(minKarma && { minKarma }),
+  };
+
+  const startDateFilter = startDate ? `AND p."postedAt" >= $(startDate) ` : '';
+  const endDateFilter = endDate ? `AND p."postedAt" <= $(endDate) ` : '';
+  const baseScoreFilter = minKarma ? `AND p."baseScore" >= $(minKarma) ` : '';
+  const showEventsFilter = showEvents === false ? 'AND p."isEvent" IS NOT TRUE' : '';
+
+  return [`${startDateFilter}${endDateFilter}${baseScoreFilter}${showEventsFilter}`, params];
```

Comment:
🙏 

I think a `[startDateFilter, endDateFilter, baseScoreFilter, showEventsFilter].join(' ')` is probably more robust to future additions/changes, since it doesn't rely on manually ensuring all-but-the-last filter have a space at the end of the string.